LayoutTests/fast/css/font-face-attribute-remove.html

```diff
@@ -0,0 +1,6 @@
+<p>PASS if it does not crash.</p>
+<font id="f" face="helvetica"></font>
+ <script>
```

Comment:
there's some extra whitespace before the `<`

Reply:
Fixed.